airflow/jobs/backfill_job.py

```diff
@@ -773,13 +774,9 @@ def _execute(self, session=None):
         # picklin'
         pickle_id = None
 
-        try:
-            from airflow.executors.dask_executor import DaskExecutor
-        except ImportError:
-            DaskExecutor = None
-
-        if not self.donot_pickle and \
-                self.executor.__class__ not in (LocalExecutor, SequentialExecutor, DaskExecutor):
+        if not self.donot_pickle and self.executor_class not in (
+            ExecutorLoader.LOCAL_EXECUTOR, ExecutorLoader.SEQUENTIAL_EXECUTOR, ExecutorLoader.DASK_EXECUTOR,
+        ):
```

Comment:
I am wondering if it would make sense to extract this tuple to something like `UNPICKLEABLE_EXECUTORS`?

Reply:
I think it would be worth doing as an attribute in the performer so that custom executors can also affect this behavior, but that would require changes to all executors. I'll create a ticket about that.

Reply:
I will create a ticket after merging in order to be able to create references to the code in the ticket